<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iSerTerm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="consoleWidget"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="sendLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="sendLabel">
          <property name="text">
           <string>Hex input:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sendLineEdit"/>
        </item>
        <item>
         <widget class="QRadioButton" name="hexRadioButton">
          <property name="text">
           <string>He&amp;x</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sendButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="decRadioButton">
          <property name="text">
           <string>Dec</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sendButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="binRadioButton">
          <property name="text">
           <string>Bin</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sendButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sendButton">
          <property name="text">
           <string>&amp;Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionSend_file"/>
    <addaction name="actionSave_log"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Se&amp;rial</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Console</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionStop_update"/>
    <addaction name="actionLocal_echo"/>
    <addaction name="actionViewSendInput"/>
    <addaction name="actionHexadecimal_view"/>
    <addaction name="separator"/>
    <addaction name="actionSet_font"/>
    <addaction name="actionSet_background_color"/>
    <addaction name="actionSet_foreground_color"/>
    <addaction name="actionConfigure_console"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="actionStop_update"/>
   <addaction name="actionViewSendInput"/>
   <addaction name="actionHexadecimal_view"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="iserterm.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="iserterm.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="iserterm.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure serial port...</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="iserterm.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="iserterm.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLocal_echo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Local &amp;echo</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionSet_font">
   <property name="icon">
    <iconset theme="font">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Set &amp;font...</string>
   </property>
  </action>
  <action name="actionSet_background_color">
   <property name="text">
    <string>Set &amp;background color...</string>
   </property>
  </action>
  <action name="actionSet_foreground_color">
   <property name="text">
    <string>Set foreground &amp;color...</string>
   </property>
  </action>
  <action name="actionStop_update">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iserterm.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop update</string>
   </property>
   <property name="toolTip">
    <string>Stop update, received data will not be lost</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionViewSendInput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iserterm.qrc">
     <normaloff>:/images/hex-input-24.png</normaloff>:/images/hex-input-24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Hexadecimal input</string>
   </property>
   <property name="shortcut">
    <string>Alt+H</string>
   </property>
  </action>
  <action name="actionSend_file">
   <property name="text">
    <string>Se&amp;nd file...</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionSave_log">
   <property name="text">
    <string>S&amp;ave log...</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionHexadecimal_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iserterm.qrc">
     <normaloff>:/images/hex-view-24.png</normaloff>:/images/hex-view-24.png</iconset>
   </property>
   <property name="text">
    <string>Hexadecimal &amp;view</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionConfigure_console">
   <property name="text">
    <string>C&amp;onfigure console..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sendLineEdit</tabstop>
  <tabstop>hexRadioButton</tabstop>
  <tabstop>decRadioButton</tabstop>
  <tabstop>binRadioButton</tabstop>
  <tabstop>sendButton</tabstop>
 </tabstops>
 <resources>
  <include location="iserterm.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="sendButtonGroup"/>
 </buttongroups>
</ui>
